installer/uninstaller.sh

```diff
@@ -43,25 +43,45 @@ helm_arr=()
 K8S_NS_ARR=()
 
 cd ../scripts
-./imm -k rm t default-tenant
-cd ../installer
+if [[ "$quiet" != "yes" ]]; then
+    echo "Uninstalling IMM. Following components will be removed:"
+    echo "*Default tenant with it's resources(models,endpoints etc.)"
+    echo "*Helm releases (mgt-api, crd etc.)"
+    echo "Are you sure you want to uninstall IMM? Y/n"
```

Comment:
If somebody just press enter here to answer yes (like users are used to), would it be 'Y' taken as default?

Reply:
Nope, I thought it would be safer to accept only fully conscious "Y" 😄